### ISSUE TITLE
Convert the categorical `reading_level` variable into 0/1 variables

### ISSUE THREAD

**Issue by jo-elimu** [elimu-ai/ml-storybook-reading-level#39]
## Description

Use [`pandas.get_dummies`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.get_dummies.html) to create one column per reading level.

## Why?

To test if this one-hot encoding technique would result in a better accuracy score.

**Comment by avithal** [comment]
data and will work on it

**Comment by saither0204** [comment]
Is this issue still existing or PR is done for it? If not I would like to be assigned.

**Comment by jo-elimu** [comment]
@saither0204 Assigned ☑️

Note that there are 4 possible reading levels: https://github.com/elimu-ai/model/blob/main/src/main/java/ai/elimu/model/v2/enums/ReadingLevel.java

**Comment by saither0204** [comment]
Since we want 4 different columns for each reading_level 

There is only one possible Multi-Output approach to measure accuracy for each column and a raw MAE would not be useful our data will be binary. So how would you like me to proceed....

**Comment by jo-elimu** [comment]
@saither0204 Thank you for that insight. Since that's the case, maybe it's better if we close this issue and stick with the current (1-column) solution?

**Comment by saither0204** [comment]
Yes I tried and could not achieve the required results and also dataset currently does not contain enough data for each reading level to train each reading level as a separate entity.
